generator/internal/codegen_utils.cc

```diff
@@ -138,14 +143,12 @@ ProcessCommandLineArgs(std::string const& parameters) {
                    [](std::pair<std::string, std::string> const& p) {
                      return p.first == "copyright_year";
                    });
-  auto current_year =
-      absl::FormatTime("%Y", absl::Now(), absl::LocalTimeZone());
   if (copyright_year != command_line_args.end() &&
       copyright_year->second.empty()) {
-    copyright_year->second = current_year;
+    copyright_year->second = CurrentCopyrightYear();
   }
   if (copyright_year == command_line_args.end()) {
-    command_line_args.emplace_back("copyright_year", current_year);
+    command_line_args.emplace_back("copyright_year", CurrentCopyrightYear());
   }
```

Comment:
Nit: I might simplify to
```
if (copyright_year == command_line_args.end()) {
  command_line_args.emplace_back("copyright_year", CurrentCopyrightYear());
} else if (copyright_year->second.empty()) {
  copyright_year->second = CurrentCopyrightYear();
}
```
to avoid the extra test, but your choice.